packages/Sandblocks-Smalltalk/SBVariantProxy.class.st

```diff
@@ -0,0 +1,122 @@
+Class {
```

Comment:
probably best at home in -Babylonian, too

Reply:
same as above